home.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>YoutTube - Custome</title>
+</head>
+<body>
+    <ul class="header">
+        <li ><a href="#">YoutTube</a></li>
+        <li style="float: right;"><a href="#">LogOut</a></li>
+    </ul>
+    <div class="side-navigation">
+        <ul class="side-nav">
+            <li><a href="#" style="font-weight: bold">Subscription Channels</a></li>
```

Comment:
- [ ] Kindly I recommend you to not use inline styles in your **html** file

Reply:
- [ ] Great job spotting 👍  this issue. But as I mentioned before, it will be great to use an explanation for each requested change. How about if you include something like: 
  - "It is considered bad practice to use inline styles in your html file because the browser doesn't understand it well"

Reply:
- [ ] Kindly I recommend you to not use inline styles in your **html** file. It is considered us a bad practice to use inline styles in HTML file because web browser doesn't understand it well